concepticondata/conceptlists/Mountain-1978-267.tsv

```diff
@@ -32,8 +32,8 @@ Mountain-1978-267-30	30	hueso	bone	32	1394	BONE
 Mountain-1978-267-31	31	sangre	blood	33	946	BLOOD
 Mountain-1978-267-32	32	corazón	heart	34	1223	HEART
 Mountain-1978-267-33	33	pene	penis	36	1222	PENIS
-Mountain-1978-267-34	34	hombre	man	38	1554	MAN
-Mountain-1978-267-35	35	varón	man	39	2106	MALE PERSON
+Mountain-1978-267-34	34	hombre	man	38	683	PERSON
+Mountain-1978-267-35	35	varón	man	39	1554	MAN
```

Comment:
I am confused now, sorry. For me, varón is like "male person". The semantic pairing would rather be "hombre" vs. "mujer", not "varón" vs. "mujer", so I think the mapping was correct. Didn't I emphasize that varón is often used to say "I have a boy" = male human being, or did I express myself wrongly? So I'd say: the original mapping was in fact correct, also consulting [PONS](https://de.pons.com/%C3%BCbersetzung/spanisch-deutsch/var%C3%B3) on this matter.